docs/src/pages/components/scrollview/demo.tsx

```diff
@@ -1,41 +1,28 @@
-import * as React from 'react';
-import { Image, ScrollView, ScrollViewProps } from '@aws-amplify/ui-react';
+import { Image, ScrollView, useTheme } from '@aws-amplify/ui-react';
+
 import { Demo } from '@/components/Demo';
-import { useScrollViewProps } from './useScrollViewProps';
-import { ScrollViewPropControls } from './ScrollViewPropControls';
-import { demoState } from '@/utils/demoState';
 
-const propsToCode = (props: ScrollViewProps) => {
-  return `<ScrollView
-  orientation="${props.orientation}"
-  height="300px"
-  width="400px"
->
+export const ScrollViewDemo = () => {
+  const {
+    tokens: { colors },
+  } = useTheme();
+  return (
+    <Demo
+      code={`
+<ScrollView width="100%" height="300px" maxWidth="580px">
```

Comment:
for the `height`, should we use `vb` instead, for responsiveness?

Reply:
`vb` is an experimental length property that is not supported by most browsers:
https://developer.mozilla.org/en-US/docs/Web/CSS/length